examples/simple_listener_example.py

```diff
@@ -21,27 +21,27 @@
 class MyCastStatusListener(CastStatusListener):
     """Cast status listener"""
 
-    def __init__(self, name, cast):
+    def __init__(self, name: str | None, cast: pychromecast.Chromecast) -> None:
         self.name = name
         self.cast = cast
 
-    def new_cast_status(self, status):
+    def new_cast_status(self, status: pychromecast.CastStatus) -> None:
         print("[", time.ctime(), " - ", self.name, "] status chromecast change:")
         print(status)
 
 
 class MyMediaStatusListener(MediaStatusListener):
     """Status media listener"""
 
-    def __init__(self, name, cast):
+    def __init__(self, name: str | None, cast: pychromecast.Chromecast) -> None:
         self.name = name
         self.cast = cast
 
-    def new_media_status(self, status):
+    def new_media_status(self, status: MediaStatus) -> None:
         print("[", time.ctime(), " - ", self.name, "] status media change:")
         print(status)
 
-    def load_media_failed(self, item, error_code):
+    def load_media_failed(self, item: int, error_code: int) -> None:
```

Comment:
What does the item represent?

Reply:
It's to identify which item of a queue failed to play, Google calls it `itemId`.

We should update the docstring for `MediaStatusListener` and maybe also rename the parameter to `queue_item` or `queue_item_id` to make it clearer what it is.